server/game/cards/01_SOR/DarthVaderDarkLordOfTheSith.ts

```diff
@@ -0,0 +1,55 @@
+import AbilityHelper from '../../AbilityHelper';
+import { LeaderUnitCard } from '../../core/card/LeaderUnitCard';
+import { Aspect, CardType, RelativePlayer, WildcardCardType } from '../../core/Constants';
+import { StateWatcherRegistrar } from '../../core/stateWatcher/StateWatcherRegistrar';
+import { CardsPlayedThisPhaseWatcher } from '../../stateWatchers/CardsPlayedThisPhaseWatcher';
+
+export default class DarthVaderDarkLordOfTheSith extends LeaderUnitCard {
+    private cardsPlayedThisPhaseWatcher: CardsPlayedThisPhaseWatcher;
+
+    protected override getImplementationId() {
+        return {
+            id: '6088773439',
+            internalName: 'darth-vader#dark-lord-of-the-sith',
+        };
+    }
+
+    protected override setupStateWatchers(registrar: StateWatcherRegistrar): void {
+        this.cardsPlayedThisPhaseWatcher = AbilityHelper.stateWatchers.cardsPlayedThisPhase(registrar, this);
+    }
+
+    protected override setupLeaderSideAbilities() {
+        this.addActionAbility({
+            title: 'Deal 1 damage to a unit and 1 damage to a base',
+            cost: [AbilityHelper.costs.abilityResourceCost(1), AbilityHelper.costs.exhaustSelf()],
+            targetResolvers: {
+                unit: { cardTypeFilter: WildcardCardType.Unit,
+                    immediateEffect: AbilityHelper.immediateEffects.conditional({ condition: (context) => this.villainyCardPlayedThisPhase(context),
+                        trueImmediateEffect: AbilityHelper.immediateEffects.damage({ amount: 1 }),
+                        falseImmediateEffect: AbilityHelper.immediateEffects.noAction() }) },
+                base: { cardTypeFilter: CardType.Base,
+                    immediateEffect: AbilityHelper.immediateEffects.conditional({ condition: (context) => this.villainyCardPlayedThisPhase(context),
+                        trueImmediateEffect: AbilityHelper.immediateEffects.damage({ amount: 1 }),
```

Comment:
Just a heads-up, in the Iden PR that just got merged I renamed these to `onTrue` and `onFalse` just to make them less verbose